repo/meta/schema/command-schema.json

```diff
@@ -1,15 +1,20 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "type": {
-    "oneOf": [
-      {
-        "type": "object",
-        "properties": {
-          "python": {
+  "oneOf": [
+    {
+      "type": "object",
+      "properties": {
+        "pip": {
+          "type": "array",
+          "items": {
             "type": "string"
-          }
+          },
+          "title": "Embedded Requirements File",
+          "description": "An array of strings representing of the requirements file to use for installing the subcommand for Pip. Each item is interpreted as a line in the requirements file."
         }
-      }
-    ]
-  }
+      },
+      "additionalProperties": false,
+      "required": ["pip"]
```

Comment:
Should this be required?  Other options (something other than pip) might be added later.